examples/decomposition/plot_incremental_pca.py

```diff
@@ -40,11 +40,14 @@
 pca = PCA(n_components=n_components)
 X_pca = pca.fit_transform(X)
 
+colors = ['navy', 'turquoise', 'darkorange']
+lw = 2
```

Comment:
It's a somewhat minor comment, but I would rather avoir adding a variable for something that is used only once. In other words, I would prefer if this value was just inlined in the call to "scatter".